lib/nolibc/musl-imported/include/sys/time.h

```diff
@@ -6,16 +6,6 @@ extern "C" {
 
 #include <uk/config.h>
 
-#define __NEED_time_t
-#define __NEED_suseconds_t
-#define __NEED_struct_timeval
```

Comment:
I think you should keep these three defines ahead and instead of including `uk/time_types.h`, include `nolibc-internal/shareddefs.h`. I have a longer explanation at the `time.h` header.

Reply:
You're right these must be kept.